sql/2021/resource-hints/pages_almanac_by_device_and_redundant_preload_font_downloads.sql

```diff
@@ -0,0 +1,77 @@
+#standardSQL
```

Comment:
General comment about the file name

I'd suggest something simpler like `redundant_preload_font_downloads.sql`. The `pages_almanac_by_device_and` part is overly descriptive.

Reply:
@rviscomi Sorry I missed this comment. The `pages_almanac_by_device`...etc was copied from some of last years chapters (specifically Markup 2020) as I was told that the queries are a good example to copy. Is there a preferred naming convention?

Reply:
No real convention, just short and simple. More about "what am I measuring" rather than "how am I measuring it". pages/almanac describe the how and device is measured on every metric so redundant.